### DISCUSSION TITLE
Internal server error

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#432] [General]
I noticed when i add user sometimes the user gets an internal server error when accessing the specific directory i gave them access to aftee seeing an authorized error despite my settings showing in that directory access as any account. It seems to be intermittent.

**Comment by rejetto** [comment]
does the user see the error before or after it has logged in?
please provide as much information as you can.
is this server on the internet?

**Comment by Stoufs** [comment]
After login qnd switching to one of the directories i gave them access to. I have to remove and readd the folder to get it working

**Comment by rejetto** [comment]
it sounds a lot like a bug that I recently fixed that is due for next release.
I'm addressing several things, so it's taking longer than I hoped.

**Reply by Stoufs** [reply]
Any eta?

**Comment by rejetto** [comment]
i can give you this preview version
from Admin > home, right click on the check-for-update, and it will ask you for the link to the update
[hfs-linux.zip](https://github.com/rejetto/hfs/files/13850523/hfs-linux.zip)
[hfs-windows.zip](https://github.com/rejetto/hfs/files/13850524/hfs-windows.zip)

**Comment by Stoufs** [comment]
Excellwnt! This version addresses that internal server error bug?

**Reply by rejetto** [reply]
of course, i'm giving it to you for this reason

**Reply by rejetto** [reply]
hopefully that it's the same problem that i fixed, and not just the same error message, we'll see
